release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -457,6 +457,12 @@ With this release, you can configure the IPv4 join subnet that is used internall
 
 For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-network_installation-config-parameters-aws[Network configuration parameters].
 
+[id="ocp-4-18-updating-oc-adm-upgrade-recommend_{context}"]
+==== Introducing the oc adm upgrade recommend command (Technology Preview)
+When updating your cluster, the `oc adm upgrade` command returns a list of the next available versions. You can use the `oc adm upgrade recommend` command to narrow down your suggestions and recommend a new target release before you launch your update. 
++
+This feature can be used for {product-title} 4.16 and newer clusters that are also connected to an update service.
```

Comment:
I don't remember we tested 4.16 and newer clusters, is it really proper for us to announce this part here? Will it leading our customers that the recommend command also available in older version oc client?

Reply:
N-2 is our standard skew ( https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/cli_tools/openshift-cli-oc#the-oc-binary ) that applies everywhere so I do not see any special reason to point it out but I guess we had a reason to put it in the RN (I do not know our conventions about what is supposed to be included in RN or not).

I do not worry about not being tested, we have not changed the API surface for the feature to not work, and the feature is TechPreview anyway (and client-side) so it is not the end of the world if it does not work in some cases.

I do not think the RN is misleading people into thinking the command is also in 4.16 client, but I guess we could stress the "...as long as you are using 4.18 `oc` client binary" 

/shrug